Formula/twoping.rb

```diff
@@ -26,35 +26,11 @@ def install
     bash_completion.install "2ping.bash_completion" => "2ping"
   end
 
-  plist_options manual: "2ping --listen", startup: true
```

Comment:
do i need to drop this from the pr? seems like `startup` was needed to launch at boot, but now it'll be on login. maybe that distinction does not matter for this formula?

Reply:
i see now that i can just leave `plist_options startup: true`